--- pp.preprocess	2022-08-28 17:55:54.697281145 +0300
+++ pp.preprocess.NEW2	2022-08-28 15:21:53.328257099 +0300
@@ -8,7 +8,7 @@
 case $1 in
 --help)
 	printf %s\\n \
-		"Usage: ${0##*/} !{printf %s "${VERSION?}"}!(denisde4ev)" \
+		"Usage: ${0##*/} !{printf %s "${_pkgver:-${VERSION?}}"}!(denisde4ev)" \
 		"  * STDIN | ${0##*/} > output -- See pp(1) for details and examples" \
 		"Syntax:" \
 		"  * Lines beginning !! are replaced with the following text evaluated" \
